File: app/src/main/java/com/meizu/mstore/sdk/demo/DemoApplication.java
package com.meizu.mstore.sdk.demo;

import android.app.Application;

import com.meizu.mstore.sdk.MzAppCenterPlatform;

public class DemoApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        MzAppCenterPlatform.Companion.init(this, "your_appKey", BuildConfig.DEBUG);
    }
}

File: kotlinApp/src/main/java/com/meizu/mstore/sdk/demo/DemoApplication.kt
package com.meizu.mstore.sdk.demo

import android.app.Application
import com.meizu.mstore.sdk.MzAppCenterPlatform

class DemoApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        MzAppCenterPlatform.init(this, "your_appKey", BuildConfig.DEBUG)
    }
}